PCBuilding.py

```diff
@@ -0,0 +1,57 @@
+import pygame, sys
+from pygame import *
+
+pygame.init()
+
+WIDTH = 800
+HEIGHT = 600
+BLACK = (0,0,0)
+WHITE = (255,255,255)
+RED = (255,0,0)
+GREEN = (0,255,0)
+BLUE = (0,0,255)
+
+counter = 0
+counterIncrease = False
+
+complexMeme = pygame.image.load('whatTheyThinkWeDo.jpg')
+simpleMeme = pygame.image.load('whatWeActuallyDo.jpg')
+
+
+
+window = pygame.display.set_mode((WIDTH, HEIGHT),0, 32)
+font = pygame.font.SysFont(None,52)
+text = font.render("",False,BLACK)
+memeTalk = text.get_rect()
+
+
+while True:
+
+    if counterIncrease == False:
```

Comment:
You could separate the counter += 1 from the if statement and put it at the top. This way you can have something making "counter = 0" when counter > x, getting rid of the need for the counterIncrease variable.